src-electron/generator/generation-engine.js

```diff
@@ -742,11 +742,12 @@ async function generateAllTemplates(
   })
 
   // Next load the partials
-  packages.forEach((singlePkg) => {
+  for (const singlePkg of packages) {
```

Comment:
Wouldn't this be easier handled by simply doing Promise.all() instead of making the load partial be non-async function?